### DISCUSSION TITLE
Logging format

### DISCUSSION THREAD

**Discussion by CasperThrane** [provectus/kafka-ui#4277] [Q&A]
Hi

Is it possible to change the format of the logging output? Either to json format? Maybe even be able to use logback-ecs-encoder. Or to standard console output, without color coding.

Br
Casper

**Comment by arjun-raviarasu** [comment]
We can update the pattern param in **logback-spring.xml** present under resources to change the format of the log. 
![image](https://github.com/provectus/kafka-ui/assets/153736466/f8f2b771-ee4f-4d86-8586-4149c49143d1)


Regards,
Arjun R
